src/server/network/webhook/webhook.cpp

```diff
@@ -12,13 +12,10 @@
 #include "server/network/webhook/webhook.h"
 #include "config/configmanager.h"
 
-// Tread no further, adventurer!
-// Go back while you still can.
+Webhook::Webhook() = default;
```

Comment:
NBC
I normally prefer "default" declarations to go in the header file, since they are harmless and give good quick understanding.